### PR TITLE
docs: added gitpod support

### PR DESCRIPTION
**Pull request by Samridhi-98**
**Description**

Changes proposed in this pull request:

-  Added `.gitpod.yml`
- Updated `README.md` with gitpod setup
- Updated `guide.md`

### CODE REVIEW

**Comment by Samridhi-98** [comment]
@derberg  @magicmatatjahu  Please review my PR.

File: docs/development/guide.md [hunk 1]
```diff
@@ -59,6 +59,12 @@ Launch the development server with the hot reloading functionality that allows a
 npm start
 ```
 
+## Launch a development environment with Gitpod
```
Comment (magicmatatjahu) [inline docs/development/guide.md:62]: You wrote that section not in appropriate place, you should write it after sentence:

> You can access the live development server at [localhost:3000](http://localhost:3000/).

Also you should change gitpod link to open `next` branch:

```
https://gitpod.io/#https://github.com/asyncapi/asyncapi-react/tree/next
```

**Review by magicmatatjahu — APPROVED** [review]
LGTM! Thanks for contribution!

**Comment by magicmatatjahu** [comment]
This is a PR with original content from this PR https://github.com/asyncapi/asyncapi-react/pull/476 but fixing some things :) Thanks also @pawlean for contribution! :)

**Pull request merged**